src/olympia/accounts/views.py

```diff
@@ -267,10 +268,18 @@ def add_api_token_to_response(response, user, set_cookie=True):
 
 class FxAConfigMixin(object):
 
+    def get_config_name(self, request):
+        return request.GET.get('config', self.DEFAULT_FXA_CONFIG_NAME)
+
+    def get_allowed_configs(self):
+        return getattr(
+            self, 'ALLOWED_FXA_CONFIGS', [self.DEFAULT_FXA_CONFIG_NAME])
+
     def get_fxa_config(self, request):
-        config_name = request.GET.get('config')
-        if config_name in getattr(self, 'ALLOWED_FXA_CONFIGS', []):
+        config_name = self.get_config_name(request)
+        if config_name in self.get_allowed_configs():
             return settings.FXA_CONFIG[config_name]
+        log.info('Using default FxA config instead of {}'.format(config_name))
```

Comment:
thanks for adding logging

![highfive](https://cloud.githubusercontent.com/assets/55398/18598555/f1641eb4-7c18-11e6-83bb-55fe78551e10.gif)